android/app/src/main/java/com/blade/MainActivity.java

```diff
@@ -1,6 +1,9 @@
 package com.razorpay.blade;
```

Comment:
Is this all the basic setup for RN gesture handler?

Can we add this to Readme of blade with a link to integrate RN gesture handler?